app/support/auto_expire_reservation.rb

```diff
@@ -14,11 +14,15 @@ def order_details
     purchased_active_order_details
   end
 
+  def earliest_allowed_time
+    Settings.reservations.timeout_period.seconds.ago
```

Comment:
I think we can get away with just `Settings.reservations.timeout_period.ago` since the settings already have the `.hours`.

Surprisingly, 
```
12.hours.seconds
=> 43200 seconds
```